lib/screens/news_screen.dart

```diff
@@ -1,7 +1,12 @@
 import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:project/controllers/article.dart';
+import 'package:project/models/article.dart';
 
 class NewsScreen extends StatelessWidget {
-  const NewsScreen({Key? key}) : super(key: key);
+  NewsScreen({Key? key}) : super(key: key);
+
+  final ArticleController articleController = Get.put(ArticleController());
```

Comment:
Here we don't care but normally we should have put it in a function called at startup so that this screen becomes a ``GetView`` and we will have direct access to the controller